demo/admin/src/products/ProductsGrid.tsx

```diff
@@ -106,6 +138,7 @@ export function ProductsGrid() {
             flex: 1,
             type: "number",
             visible: theme.breakpoints.up("md"),
+            valueFormatter: ({ value }) => intl.formatNumber(value, { style: "currency", currency: "EUR" }), // excel-column should be configured as currency-type alternatively prop below
             renderCell: ({ row }) => (typeof row.price === "number" ? <FormattedNumber value={row.price} style="currency" currency="EUR" /> : "-"),
```

Comment:
you should not need FormattedNumber AND formatNumber

Reply:
we've probably got some colliding recommendations I know of. The first is to prefer `<FormattedNumber` over `intl.formatNumber` (at least in some cases) because you don't need to add useIntl-hook. But also to prefer valueGetter or valueFormatter over renderCell (this one I'm currently not aware of, but should probably be this way, considering any kind of grid-export)

Reply:
> The first is to prefer <FormattedNumber over intl.formatNumber

yes: when both are equally possible, we perfer <FormattedNumber. But intl.formatNumber is perfectly fine, and valueFormatter can't use <FormattedNumber, so it will be intl.formatNumber in that case.